### PR TITLE
descriptions, keywords and license added to pypi

### LINKED ISSUE #168: [doc] include pypi project description
Pypi should include a minimal project description

### PR DESCRIPTION
**Pull request by ignacioct**
Closes #168 

I've taken the liberty to also add the License and the keywords from Github. It should be fine with this, but i don't know when Pypi's info is updated (pushed to master, maybe? or manually refreshed).

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,9 +1,9 @@
 [metadata]
 name = rubrix
-# description = ...
-# long_description = ...
-# keywords = ...
-# license = ...
+description = Open-source tool for tracking, exploring and evolving data for AI projects.
+long_description = Rubrix is a tool for tracking and iterating on data for artificial intelligence projects. Rubrix focuses on enabling novel, human in the loop workflows involving data scientists, subject matter experts and ML/data engineers.
+keywords = data-science natural-language-processing artificial-intelligence knowledged-graph developers-tools human-in-the-loop mlops
```
Comment (dvsrepo) [inline setup.cfg]:
```suggestion
keywords = data-science natural-language-processing text-labeling data-annotation artificial-intelligence knowledged-graph developers-tools human-in-the-loop mlops 
```

File: setup.cfg [hunk 2]
```diff
@@ -1,9 +1,9 @@
 [metadata]
 name = rubrix
-# description = ...
-# long_description = ...
-# keywords = ...
-# license = ...
+description = Open-source tool for tracking, exploring and evolving data for AI projects.
```
Comment (dvsrepo) [inline setup.cfg]:
```suggestion
description = Open-source tool for tracking, exploring, and labeling data for AI projects.
```

**Review by dvsrepo — APPROVED** [review]

File: setup.cfg (hunk 1 above)
Comment (ignacioct) [reply setup.cfg]: We can include that keyword in Github's description too, if you want.

File: setup.cfg (hunk 2 above)
Comment (ignacioct) [reply setup.cfg]: We can correct these typos en in Github's description too, if you want.

**Comment by dvsrepo** [comment]
@ignacioct yes, we can align the github description with this, thanks!

**Pull request merged**